src/tracks/playabletrack/wavetrack/ui/WaveClipUtilities.h

```diff
@@ -27,4 +29,6 @@ void fillWhere(
    std::vector<sampleCount>& where, size_t len, bool addBias, double correction,
    double t0, double sampleRate, double stretchRatio, double samplesPerPixel);
 
+std::vector<CommonTrackPanelCell::MenuItem> GetWaveClipMenuItems();
```

Comment:
I see that this is called from `WaveChannelSubView::GetMenuItems` when clicking the body of a clip, and from `WaveTrackAffordanceControls` when clicking the header of a clip. I wonder if there isn't a way of achieving a simpler flow where whether body or header is clicked, the same code is executed, only with an additional step toto select all clip if the header is hit ?

Reply:
Unfortunately, they are separate, unrelated objects. I want to keep this PR as simple as possible; I don't see much point in complicating stuff here

Reply:
FWTIW this code won't survive the replatforming

Reply:
ok